etc/proto/test_no_changes.sh

```diff
@@ -18,6 +18,10 @@ find src -regex ".*\.pb\.go" \
 )"
 
 make proto
+git status
+git add src/pps/pps.pb.go
+git commit -m "[CORE-57] protogen hack"
+git push origin PFS-57
```

Comment:
What's going on here?

Reply:
I have a weird problem with my proto generation where a specific pps proto file gets generated slightly differently (I think something to do with whitespace) locally when I run make proto. This wasn't a problem in the past, but we added a check to validate the protos match by hash a few months ago. So I work around it by having circleCI commit the proto it generated for that file. Then I pull down the commit and force push over it with this file removed. 

Its annoying, but its an easy workaround.

Reply:
Gotcha, just remember to pull it out before merging.